### DISCUSSION TITLE
Mysql restore failed because of corrupted root user password.

### DISCUSSION THREAD

**Discussion by ajay56611** [mydumper/mydumper#799] [Q&A] [answered]
Hello,

I could able to restore MYSQL-5.7 backup data into MYSQL-8.0.22 by using the mydumper(myloader) after changing the root user password and altered the auth plugin to native manually in the k8s cron job.

For some reason the root user password got corrupted and made restore failed. I've used 0.9.5 version of mydumper.

Now, i'm kind of researching on this issue to prevent this next time when we do a restore on prod. I looked at all the release notes but no where from 0.9.5 to latest versions of the mydumper it did not talk about the auth plugin. (I'm doing a review on mydumper notes to support MySQL 8 auth?  if yes i will upgrade the dumper to a version which supports this or change the docker file accordingly to support this or ?)

Please suggest me the right approach to solve this issue.

Thank you so much in Advance.

**Comment by davidducos** [comment, marked as answer]
Sorry for my late response, you should be able to use MySQL 8 authentication if you compile using MySQL 8 Libraries or use the packages that has MySQL 8 on it.
